pkg/controller/service/service_controller.go

```diff
@@ -598,6 +598,10 @@ func getNodeConditionPredicate() corelisters.NodeConditionPredicate {
 			return false
 		}
 
+		if _, hasExcludeBalancerLabel := node.Labels[constants.LabelNodeRoleExcludeBalancer]; hasExcludeBalancerLabel {
```

Comment:
I just opened #53341. We shouldn't depend on cmd/kubeadm from core controllers.

Reply:
I can fix this if food have a better location for these annotations

Reply:
Erm s/food/folks/

Reply:
We can fix this later I think.